hooks/useEscape.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from 'react';
+
+const useEscape = (onEscape: () => void) => {
+  useEffect(() => {
+    const handleEsc = (event: KeyboardEvent) => {
+      if (event.keyCode === 27) onEscape();
+    };
+    window.addEventListener('keydown', handleEsc);
+
+    return () => {
+      window.removeEventListener('keydown', handleEsc);
+    };
+  }, [onEscape]);
+};
+
+export default useEscape;
```

Comment:
Added this hook for escape key handling

Reply:
I'm surprised usehooks-ts doesn't have that one.

Reply:
I'm surprised too!